panoptes/utils/utils.py

```diff
@@ -26,9 +27,13 @@ def listify(obj):
         list:   You guessed it.
     """
     if obj is None:
-        return []
+        return list()
+    elif isinstance(obj, list):
+        return obj
+    elif isinstance(obj, (collections.abc.ValuesView, collections.abc.KeysView)):
+        return list(obj)
```

Comment:
Maybe add an
```
elif isinstance(obj, dict):
    return list(obj.values())
```
That way the dictionaries of `Events` returned by things like `Observatory.observe()` and `Observatory.autofocus_cameras()` could be passed directly to `pocs.wait_for_events(events)` without having to call their `.values()` method.

Reply:
I couldn't decide about that as it's a little non-intuitive to pass a dict and have it return just the values. A nice feature but might be surprising to a user. Then again, I'm not sure what they would reasonably expect to get back if they sent it a dict.  

I'll add it in though.